mappings/net/minecraft/world/MobSpawnerEntry.mapping

```diff
@@ -1,2 +1,39 @@
 CLASS net/minecraft/unmapped/C_edcjbljn net/minecraft/world/MobSpawnerEntry
+	FIELD f_fwsalxge nbt Lnet/minecraft/unmapped/C_hhlwcnih;
+	FIELD f_ilsksuse CODEC Lcom/mojang/serialization/Codec;
+	FIELD f_kquvrajh DATA_POOL Lcom/mojang/serialization/Codec;
+	FIELD f_swncnqkj lightData Ljava/util/Optional;
 	FIELD f_xumslcvz DEFAULT_ENTITY_ID Ljava/lang/String;
+	METHOD <init> (Lnet/minecraft/unmapped/C_hhlwcnih;Ljava/util/Optional;)V
+		ARG 1 nbt
+		ARG 2 lightData
+	METHOD equals (Ljava/lang/Object;)Z
+		ARG 1 obj
+	METHOD m_hnvblbtt getLightData ()Ljava/util/Optional;
+	METHOD m_hzqmtgsf alsoGetNbt ()Lnet/minecraft/unmapped/C_hhlwcnih;
```

Comment:
What's the mojmap name for this?

Reply:
equals is a default method ofc so um... equals
lightData (updated name) / getLightData in mojmap is called customSpawnRules which makes sense but imo lightData also works.
alsoGetNbt in mojmap is called entityToSpawn but returns the root nbtcompound (due to mojang being stupid or static final booleans), should I change it?